jbmc/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ add_subdirectory(unit)
 
 add_custom_target(java-models-library ALL
     COMMAND mvn --quiet -Dmaven.test.skip=true package
-    COMMAND cp target/core-models.jar ${CMAKE_CURRENT_SOURCE_DIR}/src/java_bytecode/library/
+    COMMAND ${CMAKE_COMMAND} -E copy target/core-models.jar ${CMAKE_CURRENT_SOURCE_DIR}/src/java_bytecode/library/
```

Comment:
Yep, we should avoid using unix-y commands in our... commands.

I'd recommend putting all the variable stuff in `"` quotes as well... I don't actually know off the top of my head how CMake handles variable expansions with whitespace in them.

Reply:
It's no worse than the _current_ situation so I'm happy for this to be merged as is - but It probably is worth trying a build on a checkout of CBMC that has spaces in its file path. On Mac/Windows it's not _that_ uncommon for this to happen.